docs/plugins/automelt.rst

```diff
@@ -1,6 +1,8 @@
 automelt
 ========
-**Tags:** `tag/fort`, `tag/auto`, `tag/items`, `tag/stockpiles`
+
+.. dfhack-tool::
+    :tags: fort, auto, items, stockpiles
```

Comment:
```suggestion
.. dfhack-tool::
    :tags: fort, auto, items, stockpiles
    :no-command:
```

Reply:
`automelt` is a command too, actually

Reply:
I had to go back and check because that didn't fit with the docs. I remember now: the command that prints out the version string. I think I chose not to document that on purpose : )

Reply:
You... are correct. I only looked for the PluginCommand call, which advertised an ability to actually do something useful.

Reply:
#2259

Reply:
Thanks, resolved